src/app/pages/StoryPage/index.jsx

```diff
@@ -78,6 +81,8 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
     version: props => <MediaPlayer {...props} assetUri={assetUri} />,
     byline: props => <StyledByline {...props} />,
     include: props => <Include {...props} />,
+    wsoj: props =>
```

Comment:
@rhenshaw56 Is there a reason `wsoj` was chosen to be the name of the preprocessor block name and not just `recommendations`?